### ISSUE TITLE
Install with npm error

### ISSUE THREAD

**Issue by jmedelliin** [intelllex/react-pdf#44]
When I try to install the library with npm, just as the readmy says, I get an incompatibility error.
![image](https://user-images.githubusercontent.com/106602828/231465428-c55146f0-1538-4c58-92e4-642e90e3322e.png)
![image](https://user-images.githubusercontent.com/106602828/231465576-2f932f60-ace1-4e1a-8f08-d524b0019bfe.png)

I already tried to change the version of react and react dom to @16 but still the same.

**Comment by yvan99** [comment]
yeah i face the same issue @jmedelliin , and this one seemed to be a good library !

**Comment by MarcusDelvecchio** [comment]
same issue here. React 18

**Comment by Heidarabdi** [comment]
i have the same issue is there way to solve it
